iam.tf

```diff
@@ -69,3 +69,24 @@ data "aws_iam_policy_document" "inline-keycloak-secretsmanager-doc" {
     )
   }
 }
+
+# allow Keycloak to publish messages to SQS
+resource "aws_iam_role_policy" "keycloak_to_alerts_concierge_user_updates_publish" {
+  name   = "keycloak-${var.environment}-alerts-concierge-user-updates-publish"
```

Comment:
Can you remove any references to downstream applications that will consume these messages? Since this is an independent public module, I want to minimize details of specific implementations.

Reply:
Done! Updated the plan.